tests/test_otdd.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import pytest
+
+from modsim2.data.loader import DMPair
+from modsim2.utils.config import load_configs
+
+
+# Fixture that returns the metric config dictionary
+@pytest.fixture(scope="module")
+def metrics_config() -> dict:
+    project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+    metrics_config_path = os.path.join(
+        project_root, "tests", "testconfig", "metrics.yaml"
+    )
+    otdd_config = load_configs(metrics_config_path)["metrics_config"]
+    # filter down to only otdd configs
+    otdd_config = {k: v for k, v in otdd_config.items() if v["class"] == "otdd"}
+    return otdd_config
+
+
+# This test checks that the distance between a dataset and itself is the expect value
+# When exact calculations are used then this will be zero
+# Approximate methods may be non-zero and these are checked against the known value
+# for the seed
+def test_cifar_otdd_same(metrics_config: dict):
```

Comment:
I'm still getting a failure on both tests even with a fresh installation of everything, so we may need to settle for the 'within a range' test for the OTDD tests!

Reply:
As discussed, the next update will change the tests to check for 'closeness' rather than equality